web_timeline/static/src/js/timeline_renderer.js

```diff
@@ -380,66 +380,87 @@ odoo.define("web_timeline.TimelineRenderer", function (require) {
          * @returns {Array}
          */
         split_groups: async function (events, group_bys) {
-            if (group_bys.length === 0) {
-                return events;
-            }
-            const groups = [];
-            groups.push({id: -1, content: _t("<b>UNASSIGNED</b>"), order: -1});
-            var seq = 1;
+            if (group_bys.length === 0) return events;
+
+            const groups = [{id: -1, content: _t("<b>UNASSIGNED</b>"), order: -1}];
+            const grouped_field = _.first(group_bys);
+            const fields = await this._rpc({
+                model: this.modelName,
+                method: "fields_get",
+                args: [[grouped_field]],
+                context: this.getSession().user_context,
+            });
+            const fieldType = fields[grouped_field].type;
+            const relation = fields[grouped_field].relation;
```

Comment:
I think this RPC call can be avoided if you use `this.fields`, which already contains metadata for fields (such as type, relation, etc.).
![image](https://github.com/user-attachments/assets/58118065-db2a-4eec-bc82-2cb768c71c6a)


```suggestion
            const fieldType = this.fields[grouped_field].type;
            const relation = this.fields[grouped_field].relation;
```

Reply:
Thank you, good catch!!